### PR TITLE
Ordering

### PR DESCRIPTION
**Pull request by LawrenceWhalen**
## Changes
  Add ability to order the reviews by any field in allReviews query
  Add error catching to prevent 500 when an incorrect field is called

## Checklist
- [x] Tests written where required
- [x] All tests passing
- [x] SimpleCov > 95%

closes #

### CODE REVIEW

File: app/controllers/graphql_controller.rb [hunk 1]
```diff
@@ -14,6 +14,8 @@ def execute
     }
     result = DateEmRateEmBeSchema.execute(query, variables: variables, context: context, operation_name: operation_name)
     render json: result
+  rescue ActiveRecord::StatementInvalid => e
+    render json: { errors: [{ message: "Invalid order_by argument for #{params['query'].split(' ')[1][0..-2]}" }], data: {} }, status: 404
```
Comment (Aferencz1987) [inline app/controllers/graphql_controller.rb:18]: Sad path for ordering review information

File: app/graphql/types/query_type.rb [hunk 2]
```diff
@@ -7,49 +7,50 @@ class QueryType < Types::BaseObject
     # Add root-level fields here.
     # They will be entry points for queries on your schema.
```
Comment (Aferencz1987) [inline app/graphql/types/query_type.rb:9]: this looks like the same thing removed and replaced

**Review by Aferencz1987 — APPROVED** [review]

**Review by ahenzler — APPROVED** [review]
great changes. everything looks good.

File: app/graphql/types/query_type.rb [hunk 3]
```diff
@@ -7,49 +7,50 @@ class QueryType < Types::BaseObject
...
+        argument :id, ID, required: true
+      end
+      def single_client(id:)
+        Client.find(id)
+      end
+
+    # REVIEWS -----------------------------------------------
+
+      field :all_reviews, [Types::ReviewType], null: false do
+        argument :order_by, String, required: false
```
Comment (ahenzler) [inline app/graphql/types/query_type.rb:41]: really like this optional argument

**Review by ahenzler — APPROVED** [review]

**Pull request merged**